docker-compose.yml

```diff
@@ -21,13 +21,19 @@ services:
        context: .
        dockerfile: Dockerfile
        args:
-         woocommerce_version: 3.1.0
+         woocommerce_version: 4.0.1
      ports:
        - "8000:80"
      restart: always
      environment:
+       WORDPRESS_DEBUG: 1
        WORDPRESS_DB_HOST: db:3306
        WORDPRESS_DB_PASSWORD: wordpress
+       WORDPRESS_CONFIG_EXTRA: |
+        define( 'WPLANG', '${WPLANG}');
+        define( 'WP_DEBUG_LOG', '/tmp/wp-errors.log');
+        define( 'WP_SITEURL', 'http://' . $$_SERVER['HTTP_HOST']);
```

Comment:
Setting `WP_SITEURL` and `WP_HOME` as per the [ngrok usage guide](https://ngrok.com/docs#wordpress)